Source/Kesmai.Server/Game/Items/Equipment/Gauntlets/Gauntlets.cs

```diff
@@ -137,6 +139,22 @@ public override int CalculateBlockingBonus(ItemEntity item)
 			return BaseArmorBonus;
 		}
 		
+		/// <summary>
+		/// Gauntlets at this time do not provide mitigation against weapons.
```

Comment:
I think we should go ahead and add the code, that way if we need to tweak gaunts later we don't also have to revisit this in order to do it. Setup the knob to turn even if we aren't currently turning it.